test/e2e/consumer_group_lag_provider_test.go

```diff
@@ -0,0 +1,56 @@
+// +build e2e
+
+/*
+ * Copyright 2021 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/types"
+	testlib "knative.dev/eventing/test/lib"
+
+	kafkatest "knative.dev/eventing-kafka-broker/test/pkg/kafka"
+	pkgtesting "knative.dev/eventing-kafka-broker/test/pkg/testing"
+)
+
+func TestConsumerGroupLagProvider(t *testing.T) {
```

Comment:
Can you use the new test framework for this e2e test? I would love to avoid adding new tests using the old test code, if possible

Reply:
Great suggestion! Let me check what I can do with it.

Reply:
I tried some stuff, but this requires some other changes to some of our helpers, that don't belong to this PR.
Also, I'm a little worried about the flakiness we've seen with the new test framework in eventing.
With that being said, I'd like to wait for this migration for now.

Reply:
Ok, then we'll do that in another PR